1-development/list_posts_by_rating/Gemfile

```diff
@@ -0,0 +1,42 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.4'
+# Use sqlite3 as the database for Active Record
+# gem 'sqlite3', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+# gem 'jbuilder', '~> 2.7'
+# Use Redis adapter to run Action Cable in production
+# gem 'redis', '~> 4.0'
+# Use Active Model has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+gem 'pg'
```

Comment:
PostgresSQL as DB

Reply:
Why?

Reply:
I believe that Postgresql is the best db to deal with more complex solutions. And for using `SQRT` postgres already has native solution. SQLite does not.